plugin/wavesurfer.minimap.js

```diff
@@ -94,7 +94,7 @@ WaveSurfer.Minimap = WaveSurfer.util.extend({}, WaveSurfer.Drawer, WaveSurfer.Dr
 
     bindWaveSurferEvents: function () {
         var my = this;
-        this.wavesurfer.on('ready', this.render.bind(this));
+        this.wavesurfer.on('ready', my.render());
```

Comment:
@entonbiba you are missing the point.

`this.wavesurfer.on('ready', someCallback)` – that's a subscription. The first parameter is the name of an event, the second parameter is a function.

What you are doing is calling the callback immediately, instead of passing it as a parameter:
`this.wavesurfer.on('ready', someCallback())`

This is equivalent to
```
someCallback();
this.wavesurfer.on('ready', undefined)
```

So if you think the render method should be called immediately, just call it without the on-ready call.